packages/runtime/src/enhancements/node/create-enhancement.ts

```diff
@@ -120,13 +122,22 @@ export function createEnhancement<DbClient extends object>(
         }
     }
 
-    // password enhancement must be applied prior to policy because it changes then length of the field
+    // password and encrypted enhancement must be applied prior to policy because it changes then length of the field
     // and can break validation rules like `@length`
     if (hasPassword && kinds.includes('password')) {
         // @password proxy
         result = withPassword(result, options);
     }
 
+    if (hasEncrypted && kinds.includes('encrypted')) {
+        if (!options.encryption) {
```

Comment:
Shall we validate the shape of `options.encryption`? Here or inside the `EncryptedHandler` constructor.

Reply:
I feel like validating inside of the handler constructor makes better sense, and its together with the rest of the `encrypted` logic